### PR TITLE
[minor] fixed the grammar

### PR DESCRIPTION
**Pull request by dhilipkumars**
[minor] fixed the grammar

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4776)

### CODE REVIEW

**Comment by chenopis** [comment]
Looks like there was some sort of networking error, so I have manually restarted the preview build.

File: docs/concepts/abstractions/pod-termination.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Since Pods represent processes running on your cluster, Kubernetes provides for
...
 1. Kubernetes updates the Pod status to reflect the time after which the Pod is to be considered "dead" (the time of the termination request plus the grace period).
 1. Kubernetes marks the Pod state as "Terminating" and stops sending traffic to the Pod.
-1. Kubernetes send a `TERM` signal to the Pod, indicating that the Pod should shut down.
+1. Kubernetes sends a `TERM` signal to the Pod, indicating that the Pod should shut down.
 1. When the grace period expires, Kubernetes issues a `SIGKILL` to any processes still running in the Pod.
 1. Kubernetes removes the Pod from the API server on the Kubernetes Master.
 
-> **Note:** The grace period is configurable; you can set your own grace period when interacting with the cluster to request termination, such as using the `kubectl delete` command. See the [Terminating a Pod]() tutorial for more information.
\ No newline at end of file
+> **Note:** The grace period is configurable; you can set your own grace period when interacting with the cluster to request termination, such as using the `kubectl delete` command. See the [Terminating a Pod]() tutorial for more information.
```
Comment (zacharysarah) [inline docs/concepts/abstractions/pod-termination.md]: @dhilipkumars 👋  Thanks for making this update! We have new [callout syntax](https://kubernetes.io/docs/home/contribute/style-guide/#note) for notes&mdash;please add a line with the correct syntax. 👍 

So:
```
**Note:** The grace period is configurable; you can set your own grace period when interacting with the cluster to request termination, such as using the `kubectl delete` command. See the [Terminating a Pod]() tutorial for more information.
{: .note}
```

Comment (zacharysarah) [inline docs/concepts/abstractions/pod-termination.md]: @dhilipkumars In the future, if you check the box allowing edits from maintainers, we can make small edits ☝️  for submitters. 
Allow edits from maintainers checkbox

Comment (dhilipkumars) [inline docs/concepts/abstractions/pod-termination.md]: done.

**Pull request merged**